demos/using-dev-build/vite.plugins.ts

```diff
@@ -0,0 +1,38 @@
+import { readFileSync } from "fs";
+import { join } from "path";
+
+export const readCanisterIds = ({
+  prefix,
```

Comment:
is prefix used?

Reply:
its a plugin / mapper that works in any project for any developer of the community.

since `using-dev-build` is an example of project for the community, I think it makes sense to provide a mapper that works all the time therefore the few unused options.

if you want me to remove anything that we do not use here, can do. Let me know.

Reply:
Yeah especially if we're gonna share it with the community I'd rather make it as simple as possible; I think having a parameter that's just unused is more confusing than anything. What do you think?

Reply:
I removed the options we don't use.